modules/core/index.js

```diff
@@ -0,0 +1,10 @@
+export { Connection } from './connection';
+export { Difference } from './difference';
+export { Entity } from './entity';
+export { Graph } from './graph';
+export { History } from './history';
+export { iD_Node as Node } from './node';
```

Comment:
ya sure, but eventually we still are reexporting them as `Node`, `Way`, `Relation`, because most of our code uses `iD.Node`. 
```Javascript
export { OSMNode as Node } from './node';
````
or do you want me to do it 
```Javascript
export { OSMNode } from './node';
````
and refactor all the iD.Node to iD.OSMNode, that can be a big task...